pandas/tests/reshape/test_wide_to_long.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import pandas as pd
+from pandas.util import testing as tm
+import pytest
+import numpy as np
+
+
+@pytest.fixture
+def buggy_wide_df():
+    #Here, we have the rare case that a column name (A) is a substring
+    #of our single stubname (PA)
+    wide_data = {'node_id': {0: 0, 1: 1, 2: 2, 3: 3, 4: 4},
+                 'A': {0: 0.80, 1: 0.0, 2: 0.25, 3: 1.0, 4: 0.81},
+                 'PA0': {0: 0.74, 1: 0.56, 2: 0.56, 3: 0.98, 4: 0.6},
+                 'PA1': {0: 0.77, 1: 0.64, 2: 0.52, 3: 0.98, 4: 0.67},
+                 'PA3': {0: 0.34, 1: 0.70, 2: 0.52, 3: 0.98, 4: 0.67}
+                }
+
+    return pd.DataFrame.from_dict(wide_data)
+
+@pytest.fixture
+def wide_df():
+    wide_df = pd.DataFrame({'famid': [1, 1, 1, 2, 2, 2, 3],
+                            'birth': [1, 2, 3, 1, 2, 3, 1],
+                            'ht1': [2.8, 2.9, 2.2, 2, 1.8, 1.9, 2.2],
+                            'ht2': [3.4, 3.8, 2.9, 3.2, 2.8, 2.4, 3.3]})
+    return wide_df
+
+
+def test_reshape_bug_fix(buggy_wide_df):
+
+    expected_long_df = pd.wide_to_long(buggy_wide_df,
+                                       stubnames=['PA'],
+                                       i = ['node_id', 'A'],
+                                       j = 'time')
+    #this used to raise an error
```

Comment:
don't need the comment, rather put a comment on the isseue for the test

Reply:
done